src/Avalonia.FreeDesktop/DBusTrayIconImpl.cs

```diff
@@ -219,13 +221,6 @@ public StatusNotifierItemDbusObj(Connection connection, ObjectPath dbusMenuPath)
             Connection = connection;
             BackingProperties.Menu = dbusMenuPath;
             BackingProperties.ToolTip = (string.Empty, Array.Empty<(int, int, byte[])>(), string.Empty, string.Empty);
-            BackingProperties.IconName = string.Empty;
```

Comment:
Why exactly are we removing those default values?

Reply:
Null values are now handled like in the original Tmds.DBus library and don't cause a NRE anymore.